pkgs/development/tools/ytt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "ytt";
+  version = "0.30.0";
+
+  src = fetchFromGitHub {
+    owner = "vmware-tanzu";
+    repo = "carvel-ytt";
+    rev = "v${version}";
+    sha256 = "0v9wp15aj4r7wif8i897zwj3c6bg41b95kk7vi3a3bzin814qn6l";
+  };
+
+  goPackagePath = "github.com/vmware-tanzu/carvel-ytt";
+
+  vendorSha256 = null;
+
+  # on v0.30.0 YAML marshalling tests fail without consequence for ytt itself
```

Comment:
It is easily possible to disable this one test?

Reply:
Yep, in fact their [developer documentation](https://github.com/vmware-tanzu/carvel-ytt/blob/develop/docs/dev.md) refers to a script which does just that. Since the script has other dependencies, I've pulled out the important line into a custom checkPhase.